### PR TITLE
Validate $TWIN

### PR DESCRIPTION
**Pull request by Trump-Wif-Hat**
# Validate [$TWIN](https://solscan.io/token/tRUmpoU2VypUDoK8CScoftzCrBvyUgFNpbFqAYm2hMg)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/trumpwifhat_sol/status/1800257015266074636?s=46
- Coingecko/ CMC URL (If available): Still working to get coin gecko set up for $TWIN

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by presi2** [comment]
Validate $TWIN

File: validated-tokens.csv
[the remark this answers is not in the document]
Comment (presi2) [reply validated-tokens.csv:1]: Validate $TWIN

**Review by presi2 — COMMENTED** [review]
Validate $TWIN

**Comment by imcryptonaire** [comment]
Validate $TWIN Trump Wif Hat

**Comment by imcryptonaire** [comment]
We need $TWIN validated!

**Comment by imcryptonaire** [comment]
Pls validate $TWIN

**Comment by SlavaGeorgiev** [comment]
* [x]   Mint address: tRUmpoU2VypUDoK8CScoftzCrBvyUgFNpbFqAYm2hMg
 * [x]   Symbol: TWIN
 * [x]   Token Decimals: 9
 * [x]   Token icon: https://tmcgcpreoxzx72pob6rfglbtrjdlxdj64gkuoy5rg25bfe2nhsoa.arweave.net/mwRhPiR183_p7g-iUywzika7jT7hlUdjsTa6EpNNPJw
 * [x]   On-chain metadata matches
 * [x]   No duplication on Jupiter's strict list

**Comment by charcutier22** [comment]
$Twin is the next big coin on #Solana

**Comment by imcryptonaire** [comment]
* [x]   Mint address: tRUmpoU2VypUDoK8CScoftzCrBvyUgFNpbFqAYm2hMg
 * [x]   Symbol: TWIN
 * [x]   Token Decimals: 9
 * [x]   Token icon: https://tmcgcpreoxzx72pob6rfglbtrjdlxdj64gkuoy5rg25bfe2nhsoa.arweave.net/mwRhPiR183_p7g-iUywzika7jT7hlUdjsTa6EpNNPJw
 * [x]   On-chain metadata matches
 * [x]   No duplication on Jupiter's strict list

File: validated-tokens.csv
Comment (presi2) [inline validated-tokens.csv:1]: * [x]   Mint address: tRUmpoU2VypUDoK8CScoftzCrBvyUgFNpbFqAYm2hMg
 * [x]   Symbol: TWIN
 * [x]   Token Decimals: 9
 * [x]   Token icon: https://tmcgcpreoxzx72pob6rfglbtrjdlxdj64gkuoy5rg25bfe2nhsoa.arweave.net/mwRhPiR183_p7g-iUywzika7jT7hlUdjsTa6EpNNPJw
 * [x]   On-chain metadata matches
 * [x]   No duplication on Jupiter's strict list

Use this template to create comments on Github

**Comment by Simpolaya** [comment]
And a validate of the TWIN we trust 🔥

**Review by presi2 — COMMENTED** [review]
* [x]   Mint address: tRUmpoU2VypUDoK8CScoftzCrBvyUgFNpbFqAYm2hMg
 * [x]   Symbol: TWIN
 * [x]   Token Decimals: 9
 * [x]   Token icon: https://tmcgcpreoxzx72pob6rfglbtrjdlxdj64gkuoy5rg25bfe2nhsoa.arweave.net/mwRhPiR183_p7g-iUywzika7jT7hlUdjsTa6EpNNPJw
 * [x]   On-chain metadata matches
 * [x]   No duplication on Jupiter's strict list

**Comment by Ayrrad** [comment]
Validate $TWIN frens, it's worth it.

**Comment by Jymfletzfur** [comment]
token tries to submit same token on 2 rows, please amend or resubmit

**Pull request closed without merging**